wdl/model/shared/src/main/scala/wdl/shared/FileSizeLimitationConfig.scala

```diff
@@ -1,8 +1,8 @@
-package cromwell.backend.wdl
+package wdl.shared
```

Comment:
oh I moved this to `cromwell.backend` in my PR but now it's looking a little more WDLy